src/widgets/helper/ChannelView.cpp

```diff
@@ -1512,15 +1518,30 @@ void ChannelView::mouseMoveEvent(QMouseEvent *event)
         this->currentMousePosition_ = event->screenPos();
     }
 
-    // is selecting
+    // check for word underneath cursor
+    const MessageLayoutElement *hoverLayoutElement =
+        layout->getElementAt(relativePos);
+
+    //selecting single characters
     if (this->isLeftMouseDown_)
```

Comment:
We don't need to do this work if we're double-clicking and `isDoubleClick_` ⇒ `isLeftMouseDown_`.

```suggestion
    if (this->isLeftMouseDown_ && !this->isDoubleClick_)
```

Reply:
i didnt add it originally because both bools appears to be mutually exclusive, when tracing `isLeftMouseDown_` in `mouseDoubleClickEvent` it never appears to be true. Should we add this to possibly future proof?

Reply:
If they're mutually exclusive, it's fine.